src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCostOneDay = 40;
+  const minDiscount = 20;
+  const maxDiscount = 50;
+  const minDaysForDiscount = 3;
+  const maxDaysForMaxDiscount = 7;
+
+  if (days < minDaysForDiscount) {
+    return rentalCostOneDay * days;
+  }
```

Comment:
You calculate `rentalCostOneDay * days` three times. Do it once and use a variable.